bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties

```diff
@@ -462,6 +462,13 @@ status-bridge-offline = Bridge is not connected to Netatmo API
 device-not-connected = Thing is not reachable
 data-over-limit = Data seems quite old
 request-time-out = Request timed out - will attempt reconnection later
+deserialization-unknow = Deserialization lead to an unknown code
```

Comment:
Noticed typo here in Crowdin translation PR: unknow.

Reply:
Well spotted ! 
I had this typo correction in the other PR I have in review